homework/home2.1/UserStatistics.java

```diff
@@ -0,0 +1,69 @@
+import java.io.*;
```

Comment:
For Java version we can reuse most of the comments from C++ version. We should have single responsibility for the class UserStatistics, business logic should be moved to the higher level. Some arguments handling improvements are also possible.